### ISSUE TITLE
Object.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by Stanzilla** [mpeterson2/save-session#96]
Requiring `$` from `atom` is no longer supported.
If you are using `space-pen`, please require `$` from `atom-space-pen-views`. Otherwise require `jquery` instead:
  `{$} = require 'atom-space-pen-views'`
  or
  `$ = require 'jquery'`
Add `"atom-space-pen-views": "^2.0.3"` to your package dependencies.
Or add `"jquery": "^2"` to your package dependencies.
```
Object.Object.defineProperty.get (C:\Users\Benjamin\AppData\Local\atom\app-0.189.0\resources\app\exports\atom.js:43:9)
Object.<anonymous> (C:\Users\Benjamin\.atom\packages\save-session\lib\save-session.coffee:1:1)
```

**Comment by anthonyraymond** [comment]
Deplicate of #92

**Comment by mpeterson2** [comment]
Should be fixed in `0.13.0`
